src/Main.java

```diff
@@ -0,0 +1,34 @@
+public class Main {
+
+    public static void main(String[] args) {
+
+        int zahl1=333, zahl2=55;
```

Comment:
bitte nicht mehrere Variablen in einer Zeile anlegen 🙏 
lieber 
```
int zahl1 = 333;
int zahl2 = 55;
```

ebenso hat es keinen vorteil variablen vor der ersten verwendung zu deklarieren (also ohne wert anzulegen) da ists besser du schreibst da wo summe & co s erste mal verwendet wird nochmal den datentyp davor. 

Versuch auch bei der ganzzahligen variante den Platzhalter zu verwenden, für ganzzahlige Werte allerdings. 

Und vor der Abgabe bitte nochmal den code formattieren (Strg+Alt+L im IntelliJ)